Content Package Converter Exception org/apache/jackrabbit/api/ReferenceBinary

With version 1.0.0 I see the following exception when executing `analyse-project`:

```
[ERROR] Failed to execute goal com.adobe.aem:aemanalyser-maven-plugin:1.0.0:project-analyse (analyser) on project kaw-website-complete-package: Content Package Converter Exception Content Package Converter Exception org/apache/jackrabbit/api/ReferenceBinary: org.apache.jackrabbit.api.ReferenceBinary -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal com.adobe.aem:aemanalyser-maven-plugin:1.0.0:project-analyse (analyser) on project kaw-website-complete-package: Content Package Converter Exception Content Package Converter Exception org/apache/jackrabbit/api/ReferenceBinary
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:215)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: org.apache.maven.plugin.MojoExecutionException: Content Package Converter Exception Content Package Converter Exception org/apache/jackrabbit/api/ReferenceBinary
    at com.adobe.aem.analyser.mojos.AemAnalyseMojo.convertContentPackages (AemAnalyseMojo.java:227)
    at com.adobe.aem.analyser.mojos.AemAnalyseMojo.execute (AemAnalyseMojo.java:203)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: java.io.IOException: Content Package Converter Exception org/apache/jackrabbit/api/ReferenceBinary
    at com.adobe.aem.analyser.AemPackageConverter.convert (AemPackageConverter.java:124)
    at com.adobe.aem.analyser.mojos.AemAnalyseMojo.convertContentPackages (AemAnalyseMojo.java:225)
    at com.adobe.aem.analyser.mojos.AemAnalyseMojo.execute (AemAnalyseMojo.java:203)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: java.lang.NoClassDefFoundError: org/apache/jackrabbit/api/ReferenceBinary
    at org.apache.sling.feature.cpconverter.handlers.XmlConfigurationEntryHandler$JcrConfigurationHandler.onJcrRootElement (XmlConfigurationEntryHandler.java:74)
    at org.apache.sling.feature.cpconverter.shared.AbstractJcrNodeParser.onJcrRootNode (AbstractJcrNodeParser.java:64)
    at org.apache.sling.feature.cpconverter.shared.AbstractJcrNodeParser.startElement (AbstractJcrNodeParser.java:57)
    at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.startElement (AbstractSAXParser.java:509)
    at com.sun.org.apache.xerces.internal.parsers.AbstractXMLDocumentParser.emptyElement (AbstractXMLDocumentParser.java:182)
    at com.sun.org.apache.xerces.internal.impl.dtd.XMLDTDValidator.emptyElement (XMLDTDValidator.java:766)
    at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanStartElement (XMLDocumentFragmentScannerImpl.java:1339)
    at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl$ContentDriver.scanRootElementHook (XMLDocumentScannerImpl.java:1289)
    at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next (XMLDocumentFragmentScannerImpl.java:3132)
    at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl$PrologDriver.next (XMLDocumentScannerImpl.java:852)
    at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next (XMLDocumentScannerImpl.java:602)
    at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument (XMLDocumentFragmentScannerImpl.java:505)
    at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse (XML11Configuration.java:842)
    at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse (XML11Configuration.java:771)
    at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse (XMLParser.java:141)
    at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse (AbstractSAXParser.java:1213)
    at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse (SAXParserImpl.java:643)
    at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse (SAXParserImpl.java:327)
    at javax.xml.parsers.SAXParser.parse (SAXParser.java:195)
    at org.apache.sling.feature.cpconverter.shared.AbstractJcrNodeParser.parse (AbstractJcrNodeParser.java:49)
    at org.apache.sling.feature.cpconverter.handlers.XmlConfigurationEntryHandler.parseConfiguration (XmlConfigurationEntryHandler.java:45)
    at org.apache.sling.feature.cpconverter.handlers.AbstractConfigurationEntryHandler.handle (AbstractConfigurationEntryHandler.java:74)
    at org.apache.sling.feature.cpconverter.ContentPackage2FeatureModelConverter.process (ContentPackage2FeatureModelConverter.java:455)
    at org.apache.sling.feature.cpconverter.ContentPackage2FeatureModelConverter.onFile (ContentPackage2FeatureModelConverter.java:461)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:110)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:89)
    at org.apache.sling.feature.cpconverter.ContentPackage2FeatureModelConverter.processSubPackage (ContentPackage2FeatureModelConverter.java:359)
    at org.apache.sling.feature.cpconverter.handlers.ContentPackageEntryHandler.processSubPackage (ContentPackageEntryHandler.java:31)
    at org.apache.sling.feature.cpconverter.handlers.AbstractContentPackageHandler.handle (AbstractContentPackageHandler.java:93)
    at org.apache.sling.feature.cpconverter.ContentPackage2FeatureModelConverter.process (ContentPackage2FeatureModelConverter.java:455)
    at org.apache.sling.feature.cpconverter.ContentPackage2FeatureModelConverter.onFile (ContentPackage2FeatureModelConverter.java:461)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:110)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:89)
    at org.apache.sling.feature.cpconverter.ContentPackage2FeatureModelConverter.secondPass (ContentPackage2FeatureModelConverter.java:277)
    at org.apache.sling.feature.cpconverter.ContentPackage2FeatureModelConverter.convert (ContentPackage2FeatureModelConverter.java:225)
    at com.adobe.aem.analyser.AemPackageConverter.convert (AemPackageConverter.java:122)
    at com.adobe.aem.analyser.mojos.AemAnalyseMojo.convertContentPackages (AemAnalyseMojo.java:225)
    at com.adobe.aem.analyser.mojos.AemAnalyseMojo.execute (AemAnalyseMojo.java:203)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
Caused by: java.lang.ClassNotFoundException: org.apache.jackrabbit.api.ReferenceBinary
    at org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy.loadClass (SelfFirstStrategy.java:50)
    at org.codehaus.plexus.classworlds.realm.ClassRealm.unsynchronizedLoadClass (ClassRealm.java:271)
    at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass (ClassRealm.java:247)
    at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass (ClassRealm.java:239)
    at org.apache.sling.feature.cpconverter.handlers.XmlConfigurationEntryHandler$JcrConfigurationHandler.onJcrRootElement (XmlConfigurationEntryHandler.java:74)
    at org.apache.sling.feature.cpconverter.shared.AbstractJcrNodeParser.onJcrRootNode (AbstractJcrNodeParser.java:64)
    at org.apache.sling.feature.cpconverter.shared.AbstractJcrNodeParser.startElement (AbstractJcrNodeParser.java:57)
    at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.startElement (AbstractSAXParser.java:509)
    at com.sun.org.apache.xerces.internal.parsers.AbstractXMLDocumentParser.emptyElement (AbstractXMLDocumentParser.java:182)
    at com.sun.org.apache.xerces.internal.impl.dtd.XMLDTDValidator.emptyElement (XMLDTDValidator.java:766)
    at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanStartElement (XMLDocumentFragmentScannerImpl.java:1339)
    at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl$ContentDriver.scanRootElementHook (XMLDocumentScannerImpl.java:1289)
    at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl$FragmentContentDriver.next (XMLDocumentFragmentScannerImpl.java:3132)
    at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl$PrologDriver.next (XMLDocumentScannerImpl.java:852)
    at com.sun.org.apache.xerces.internal.impl.XMLDocumentScannerImpl.next (XMLDocumentScannerImpl.java:602)
    at com.sun.org.apache.xerces.internal.impl.XMLDocumentFragmentScannerImpl.scanDocument (XMLDocumentFragmentScannerImpl.java:505)
    at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse (XML11Configuration.java:842)
    at com.sun.org.apache.xerces.internal.parsers.XML11Configuration.parse (XML11Configuration.java:771)
    at com.sun.org.apache.xerces.internal.parsers.XMLParser.parse (XMLParser.java:141)
    at com.sun.org.apache.xerces.internal.parsers.AbstractSAXParser.parse (AbstractSAXParser.java:1213)
    at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl$JAXPSAXParser.parse (SAXParserImpl.java:643)
    at com.sun.org.apache.xerces.internal.jaxp.SAXParserImpl.parse (SAXParserImpl.java:327)
    at javax.xml.parsers.SAXParser.parse (SAXParser.java:195)
    at org.apache.sling.feature.cpconverter.shared.AbstractJcrNodeParser.parse (AbstractJcrNodeParser.java:49)
    at org.apache.sling.feature.cpconverter.handlers.XmlConfigurationEntryHandler.parseConfiguration (XmlConfigurationEntryHandler.java:45)
    at org.apache.sling.feature.cpconverter.handlers.AbstractConfigurationEntryHandler.handle (AbstractConfigurationEntryHandler.java:74)
    at org.apache.sling.feature.cpconverter.ContentPackage2FeatureModelConverter.process (ContentPackage2FeatureModelConverter.java:455)
    at org.apache.sling.feature.cpconverter.ContentPackage2FeatureModelConverter.onFile (ContentPackage2FeatureModelConverter.java:461)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:110)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:89)
    at org.apache.sling.feature.cpconverter.ContentPackage2FeatureModelConverter.processSubPackage (ContentPackage2FeatureModelConverter.java:359)
    at org.apache.sling.feature.cpconverter.handlers.ContentPackageEntryHandler.processSubPackage (ContentPackageEntryHandler.java:31)
    at org.apache.sling.feature.cpconverter.handlers.AbstractContentPackageHandler.handle (AbstractContentPackageHandler.java:93)
    at org.apache.sling.feature.cpconverter.ContentPackage2FeatureModelConverter.process (ContentPackage2FeatureModelConverter.java:455)
    at org.apache.sling.feature.cpconverter.ContentPackage2FeatureModelConverter.onFile (ContentPackage2FeatureModelConverter.java:461)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:110)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:102)
    at org.apache.sling.feature.cpconverter.vltpkg.BaseVaultPackageScanner.traverse (BaseVaultPackageScanner.java:89)
    at org.apache.sling.feature.cpconverter.ContentPackage2FeatureModelConverter.secondPass (ContentPackage2FeatureModelConverter.java:277)
    at org.apache.sling.feature.cpconverter.ContentPackage2FeatureModelConverter.convert (ContentPackage2FeatureModelConverter.java:225)
    at com.adobe.aem.analyser.AemPackageConverter.convert (AemPackageConverter.java:122)
    at com.adobe.aem.analyser.mojos.AemAnalyseMojo.convertContentPackages (AemAnalyseMojo.java:225)
    at com.adobe.aem.analyser.mojos.AemAnalyseMojo.execute (AemAnalyseMojo.java:203)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at sun.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:498)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
[ERROR]
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
```

Some dependency seems to be missing.
Also the exception message looks weird `Content Package Converter Exception Content Package Converter Exception org/apache/jackrabbit/api/ReferenceBinary`. Seems that the messages should be reworked in `com.adobe.aem.analyser.AemPackageConverter.convert` and `com.adobe.aem.analyser.mojos.AemAnalyseMojo.convertContentPackages`

--- a/aemanalyser-maven-plugin/pom.xml
+++ b/aemanalyser-maven-plugin/pom.xml
@@ -277,7 +277,6 @@ governing permissions and limitations under the License.
       <groupId>org.apache.sling</groupId>
       <artifactId>org.apache.sling.commons.osgi</artifactId>
       <version>2.3.0</version>
-      <scope>provided</scope>
      </dependency>
     <dependency>
       <groupId>org.apache.sling</groupId>

--- a/aemanalyser-maven-plugin/pom.xml
+++ b/aemanalyser-maven-plugin/pom.xml
@@ -103,33 +103,12 @@ governing permissions and limitations under the License.
       <artifactId>maven-artifact</artifactId>
       <version>${maven.version}</version>
     </dependency>
-    <!-- Plexus is currently needed for cpconverter -->
-    <dependency>
-      <groupId>org.codehaus.plexus</groupId>
-      <artifactId>plexus-archiver</artifactId>
-      <version>4.2.5</version>
-    </dependency>
-    <dependency>
-      <groupId>org.codehaus.plexus</groupId>
-      <artifactId>plexus-utils</artifactId>
-      <version>3.3.0</version>
-    </dependency>
-    <dependency>
-      <groupId>org.codehaus.plexus</groupId>
-      <artifactId>plexus-io</artifactId>
-      <version>3.2.0</version>
-    </dependency>
 
     <!-- Feature Model -->
     <dependency>
       <groupId>org.apache.sling</groupId>
       <artifactId>org.apache.sling.feature</artifactId>
       <version>1.2.22</version>
-    </dependency>
-    <dependency>
-      <groupId>org.osgi</groupId>
-      <artifactId>osgi.core</artifactId>
-      <version>7.0.0</version>
     </dependency>
     <dependency>
       <groupId>org.apache.geronimo.specs</groupId>
@@ -154,7 +133,7 @@ governing permissions and limitations under the License.
     <dependency>
       <groupId>org.apache.felix</groupId>
       <artifactId>org.apache.felix.cm.json</artifactId>
-      <version>1.0.6</version>    
+      <version>1.0.6</version>
     </dependency>
     <dependency>
       <groupId>org.osgi</groupId>
@@ -183,8 +162,111 @@ governing permissions and limitations under the License.
     </dependency>
     <dependency>
       <groupId>org.apache.sling</groupId>
-      <artifactId>org.apache.sling.contentparser.api</artifactId>
-      <version>2.0.0</version>
+      <artifactId>org.apache.sling.feature.cpconverter</artifactId>
+      <version>1.1.0</version>
+    </dependency>
+    
+    <!-- START: Transitive but hidden (due to scope "provided") dependencies of cpconverter, update whenever you update cpconverter
+                There is not yet a way to enforce this being complete (https://issues.apache.org/jira/browse/MENFORCER-385)
+     -->
+    <!--declared before
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-simple</artifactId>
+    </dependency>
+    -->
+    <!-- not used outside of CLI
+    <dependency>
+      <groupId>info.picocli</groupId>
+      <artifactId>picocli</artifactId>
+      <version>${picocli.version}</version>
+    </dependency>
+    -->
+    <dependency>
+      <groupId>org.apache.jackrabbit.vault</groupId>
+      <artifactId>org.apache.jackrabbit.vault</artifactId>
+      <version>3.4.10</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.jackrabbit</groupId>
+      <artifactId>jackrabbit-spi-commons</artifactId>
+      <version>2.18.4</version>
+    </dependency>
+    <dependency>
+      <groupId>javax.jcr</groupId>
+      <artifactId>jcr</artifactId>
+      <version>2.0</version>
+    </dependency>
+    <dependency>
+      <groupId>commons-io</groupId>
+      <artifactId>commons-io</artifactId>
+      <version>2.8.0</version>
+    </dependency>
+    <!-- defined before 
+    <dependency>
+      <groupId>org.apache.sling</groupId>
+      <artifactId>org.apache.sling.feature</artifactId>
+      <version>1.2.22</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.sling</groupId>
+      <artifactId>org.apache.sling.feature.extension.apiregions</artifactId>
+      <version>1.2.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.sling</groupId>
+      <artifactId>org.apache.sling.repoinit.parser</artifactId>
+      <version>1.6.8</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.felix</groupId>
+      <artifactId>org.apache.felix.converter</artifactId>
+      <version>1.0.14</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.felix</groupId>
+      <artifactId>org.apache.felix.utils</artifactId>
+      <version>1.11.4</version>
+    </dependency>
+    <dependency>
+      <groupId>org.osgi</groupId>
+      <artifactId>org.osgi.util.function</artifactId>
+      <version>1.0.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.geronimo.specs</groupId>
+      <artifactId>geronimo-json_1.1_spec</artifactId>
+      <version>1.2</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.johnzon</groupId>
+      <artifactId>johnzon-core</artifactId>
+      <version>1.2.3</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.felix</groupId>
+      <artifactId>org.apache.felix.cm.json</artifactId>
+      <version>1.0.6</version>
+    </dependency>-->
+    <dependency>
+      <groupId>org.osgi</groupId>
+      <artifactId>org.osgi.framework</artifactId>
+      <version>1.9.0</version>
+    </dependency>
+    <!-- defined before 
+    <dependency>
+      <groupId>org.apache.felix</groupId>
+      <artifactId>org.apache.felix.configadmin</artifactId>
+      <version>1.9.16</version>
+    </dependency>-->
+    <dependency>
+      <groupId>org.apache.jackrabbit</groupId>
+      <artifactId>jackrabbit-api</artifactId>
+      <version>2.18.4</version>
     </dependency>
     <dependency>
       <groupId>org.apache.sling</groupId>
@@ -193,15 +275,21 @@ governing permissions and limitations under the License.
     </dependency>
     <dependency>
       <groupId>org.apache.sling</groupId>
-      <artifactId>org.apache.sling.feature.cpconverter</artifactId>
-      <version>1.1.0</version>
-    </dependency>
-    
+      <artifactId>org.apache.sling.commons.osgi</artifactId>
+      <version>2.3.0</version>
+      <scope>provided</scope>
+     </dependency>
     <dependency>
-      <groupId>javax.jcr</groupId>
-      <artifactId>jcr</artifactId>
-      <version>2.0</version>
+      <groupId>org.apache.sling</groupId>
+      <artifactId>org.apache.sling.contentparser.api</artifactId>
+      <version>2.0.0</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.sling</groupId>
+      <artifactId>org.apache.sling.contentparser.json</artifactId>
+      <version>2.0.0</version>
+    </dependency>
+    <!-- END: Transitive but hidden (due to scope "provided") dependencies of cpconverter -->
 
     <!-- Test -->
     <dependency>

